package.json

```diff
@@ -1,37 +1,40 @@
-{
-  "name": "final-project",
-  "version": "1.0.0",
-  "description": "CS 362 Final Project",
-  "scripts": {
-    "build": "webpack",
-    "start": "webpack serve --open --port 3000",
-    "cy:open": "cypress open",
-    "cy:run": "cypress run",
-    "test:unit": "jest u-",
-    "test:integration": "jest i-",
-    "test:e2e": "start-test start http://localhost:3000 cy:run"
-  },
-  "devDependencies": {
-    "@testing-library/dom": "^9.3.4",
-    "@testing-library/jest-dom": "^6.4.2",
-    "css-loader": "^6.7.4",
-    "cypress": "^13.6.6",
-    "html-webpack-plugin": "^5.5.1",
-    "jest": "^29.7.0",
-    "jest-environment-jsdom": "^29.7.0",
-    "start-server-and-test": "^2.0.3",
-    "style-loader": "^3.3.3",
-    "webpack": "^5.83.1",
-    "webpack-cli": "^5.1.1",
-    "webpack-dev-server": "^4.15.0"
-  },
-  "jest": {
-    "testEnvironment": "jsdom",
-    "moduleNameMapper": {
-      "\\.css$": "<rootDir>/src/__mocks__/styleMock.js"
-    }
-  },
-  "dependencies": {
-    "whatwg-fetch": "^3.6.20"
-  }
-}
+{
+  "name": "final-project",
+  "version": "1.0.0",
+  "description": "CS 362 Final Project",
+  "scripts": {
+    "build": "webpack",
+    "start": "webpack serve --open --port 3000",
+    "cy:open": "cypress open",
+    "cy:run": "cypress run",
+    "test": "jest",
```

Comment:
If we define a general test script, it should probably also include the Cypress tests. Could you update this to call both or remove it please?

Reply:
?

Reply:
You defined
```json
"test": "jest",
```
here. I'm saying if we are going to add the ability to run `npm test`, it should probably execute all the tests, including the Cypress ones. I think it would be better to just remove this line from the `scripts` object. 

Alternatively, one of the lecture repos had an additional dependency to allow running several commands from one npm `script`. I think it was called `npm-run-all` or something? We'd have to track it down, but then we could set `npm test` to execute both `jest` and `cypress run`.